bundle/manifests/container-jfr-operator.clusterserviceversion.yaml

```diff
@@ -126,7 +126,7 @@ spec:
                 - --upstream=http://127.0.0.1:8080/
                 - --logtostderr=true
                 - --v=10
-                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
+                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
I can definitely remove it, but I haven't figured out how to make it run in Minikube without this fix. I just tried removing it and I get the same error again (Error: container has runAsNonRoot and image will run as root)

Reply:
That's interesting. I wonder if this is a CRC/Minikube difference or if this is more fundamental to OpenShift/Kubernetes.

Reply:
I was hoping we'd have a workaround for this by now. We may have to look into disabling the metrics endpoint temporarily, which would allow us to remove kube-rbac-proxy for now.

Reply:
I'll submit a PR to disable the metrics endpoint and kube-rbac-proxy until #180 is fixed.